lib/cocoapods/command/lib/lint.rb

```diff
@@ -25,7 +25,7 @@ def self.options
             ['--use-static-frameworks', 'Lint uses static frameworks during installation'],
             ["--sources=#{Pod::TrunkSource::TRUNK_REPO_URL}", 'The sources from which to pull dependent pods ' \
               "(defaults to #{Pod::TrunkSource::TRUNK_REPO_URL}). Multiple sources must be comma-delimited"],
-            ['--platforms=ios,macos', 'Lint against specific platforms (defaults to all platforms supported by the ' \
+            ['--platforms=ios,macos,visionos', 'Lint against specific platforms (defaults to all platforms supported by the ' \
```

Comment:
I think this is an example? maybe its missing `tvos`, `watchos`, but not for this PR.